GNUmakefile

```diff
@@ -34,22 +141,41 @@ mycfgdir = $(SYSCFG)/obs/services
 LIST_PY_FILES=git ls-tree --name-only -r HEAD | grep '\.py$$'
 PY_FILES=$(shell $(LIST_PY_FILES))
 
+ALL_PYLINT2 = pylint-2.7 pylint2.7 pylint
+ALL_PYLINT3 = pylint-3.4 pylint3.4 pylint-3.5 pylint3.5 pylint-3.6 pylint3.6
+
+PYLINT2 = $(call first_in_path_opt,$(ALL_PYLINT2))
+PYLINT3 = $(call first_in_path_opt,$(ALL_PYLINT3))
+
 default: check
 
-.PHONY: check
-check: pep8 test
+.PHONY: check check_all
+check: pep8 flake8 pylint test # test3
```

Comment:
`flake8` also verifies `pep8`, right? so you are running `pep8` twice. :thinking: